### PR TITLE
Jhalackova smo 254

### PR DESCRIPTION
**Pull request by lvicoun**
### Description

Describe the overall goals of this pull request.

### Are there any relevant issues/feature requests?

* bsc#...
* https://jira.suse.com/browse/SMO-254

### Is this (based on) existing content?

* old xml:id...
* old URL...

### CODE REVIEW

File: tasks/transactional-update-automatic-updates.xml [hunk 1]
```diff
@@ -33,8 +33,125 @@ in the assembly -->
...
+          <screen>
+RELAYHOST=“mailx”
+          </screen>
+          </listitem>
+      </varlistentry>
+      <varlistentry>
+        <term>MAILX_OPTIONS</term>
+        <listitem>
+          <para>
+            Specify necessary options to ensure that the notification mail will be accepted by the mail provider.
```
Comment (dariavladykina) [inline tasks/transactional-update-automatic-updates.xml:147]:
```suggestion
            Specify necessary options to ensure that the mail provider will accept the notification mail.
``

**Review by dariavladykina — COMMENTED** [review]
Hi, please see some suggestions. Thanks!

**Comment by lvicoun** [comment]
[transactional-updates_en.pdf](https://github.com/SUSE/doc-modular/files/14904134/transactional-updates_en.pdf)

**Comment by lkocman** [comment]
Looks good. I do recommend to also check with @laenion

**Pull request merged**